conf.yaml

```
@@ -1430,6 +1430,21 @@ contents:
     -
         title:      Enterprise Search
         sections:
+          -
+            title:      Enterprise Search Guide
+            prefix:     en/enterprise-search
+            index:      workplace-search-docs/index.asciidoc
```

Alternatively, I suggest you merge https://github.com/elastic/enterprise-search-pubs/pull/479 then this should work:
```suggestion
            index:      enterprise-search-docs/index.asciidoc
```

---

This answers my question, thanks Lisa, we will proceed.